### ISSUE TITLE
startsWith does not work in Internet Explorer

### ISSUE THREAD

**Issue by achristie** [ReactTraining/react-router#3921]
## Version
4.0.0

## Steps to reproduce
In Link.js:
```
var pathIsActive = function pathIsActive(to, pathname, activeOnlyWhenExact) {
  return activeOnlyWhenExact ? pathname === to : pathname.startsWith(to);
};
```
## Expected Behavior
exact path matching works in IE

## Actual Behavior
internet explorer doesn't know startsWith method so an exception is thrown

## Misc
I could put in a PR for this but i'm not sure how you want to fix (polyfill startsWith or use some other string manipulation)

**Comment by timdorr** [comment]
This is [added by something like `babel-polyfill`](https://github.com/zloirock/core-js/blob/master/library/modules/es6.string.starts-with.js) and is supported with Edge. Not sure if we want to fix this ourselves or rely on polyfills. I'm ambivalent about it.

**Comment by hunterc** [comment]
Could use a regex instead of `startsWith`?

**Comment by timdorr** [comment]
A PR for that would be good, if you don't mind!

**Comment by hunterc** [comment]
Away from computer at the moment but either of these should work:
```javascript
// a
return activeOnlyWhenExact ? pathname === to : pathname.substr(0, to.length) === to;

//b
return activeOnlyWhenExact ? pathname === to : new RegExp('^' + to, 'i').test(pathname);
```

**Comment by achristie** [comment]
looks like it is already fixed with 1be1ed7
